mage_ai/data_preparation/templates/data_loaders/qdrant.py

```diff
@@ -0,0 +1,26 @@
4, -0.04869353026151657, -0.0026064272969961166, 0.06487143039703369, 0.023984888568520546, 0.04395412653684616, 0.00408533588051796, 0.00381131237372756, 0.012689616531133652, -0.04801182448863983, 0.0011923470301553607, 0.046086665242910385, -0.023937679827213287, 0.16097281873226166, 0.14156579971313477, -0.03770972043275833, -0.012182109989225864, -0.020724540576338768, -0.06617094576358795, 0.07196692377328873, 0.0205035712569952, 0.1300734430551529, -0.03884392976760864, -0.020954817533493042, -0.0038270142395049334, 0.15240830183029175, -0.10839994251728058, 0.044988684356212616, -0.04287152364850044, 0.05636658892035484, -0.013559381477534771, -0.009531035088002682, -0.041882168501615524, 0.031027361750602722, 0.02470562793314457, 0.027915265411138535, 0.01717858761548996, -0.007591662462800741, -0.04226287454366684, 0.025460656732320786, -0.031810931861400604, -0.04625505208969116, -0.09031020849943161, 0.04628509655594826, -0.015553148463368416, 0.15203221142292023, -0.06094389781355858, 0.046981461346149445, 0.008619267493486404, 0.011087550781667233, -0.1421470046043396, 0.007872059009969234, 0.015657292678952217, -0.032896656543016434, 0.026725895702838898, -0.07641413062810898, 0.01612001098692417, -0.005193490069359541, -0.0627613291144371, -0.029858682304620743, 0.032059211283922195, 0.06676747649908066, 0.033120762556791306, 0.0727493017911911, 0.02020578272640705, 0.0686301589012146, 0.02969614416360855, 0.000363966595614329, -0.06517715007066727, -0.01167922094464302, 0.018274618312716484, 0.021434752270579338, 8.407879067817703e-05, -1.2356120571155316e-08, 0.05506284534931183, 0.030911119654774666, -0.009665950201451778, 0.010878344066441059, 0.07441815733909607, -0.01598217338323593, -0.053313810378313065, 0.06232559680938721, 0.04780436307191849, 0.026952575892210007, -0.015522753819823265, -0.039026252925395966, 0.0653449296951294, 0.021378427743911743, -0.016750624403357506, 0.05741793289780617, 0.007025088649243116, -0.04144252464175224, -0.052505042403936386, 0.00684284046292305, -0.07589447498321533, 0.011971106752753258, 0.041498783975839615, -0.030690189450979233, 0.05127107724547386, 0.06320663541555405, -0.06651830673217773, -0.027655906975269318, 0.042134854942560196, 0.028250088915228844, -0.031811054795980453, 0.0015333837363868952, -0.0824475809931755, 0.0578920915722847, -0.06556150317192078, 0.038768187165260315, 0.017430715262889862, 8.571466423745733e-06, 0.07248830050230026, -0.007315118331462145, -0.010102540254592896, 0.09363652020692825, 0.012293421663343906, 0.02640296332538128, -0.04357669875025749, -0.024379098787903786, 0.0040420424193143845, -0.1453550010919571, 0.04217945784330368, -0.07404214143753052, 0.05550963059067726, -0.05129440128803253, -0.025329122319817543, 0.02949410118162632, 0.07660789787769318, 0.0492815300822258, -0.02430989220738411, 0.05559298023581505, -0.04427865520119667, 0.09796113520860672, 0.06593780219554901, 0.048077549785375595, 0.014259702526032925, 0.01452113501727581]
```

Comment:
is this for testing?

Reply:
Yes, it is tested for my own data records. I updated it to contain only 6 dimentions as a mock example in template for customer.